quinn/src/connection.rs

```diff
@@ -409,6 +410,14 @@ where
             .peer_identity()
     }
 
+    /// The peer's QUIC address
```

Comment:
I think "QUIC address" is a very misleading term for this. All it is is a stable ID, and it should be documented as such.

Reply:
Good point; how does "A static identifier for this connection" sound instead?

Reply:
I prefer "stable" to "static", personally.